alarm/binance_utils.py

```diff
@@ -24,13 +24,12 @@ def get_binance_valid_trade_pairs_with_base_quote_assets():
 
 
 def format_trade_pair_for_message(trade_pair):
-    binance_valid_trade_pairs_with_base_quote_assets = get_binance_valid_trade_pairs_with_base_quote_assets()
+    trade_pairs_with_base_quote_assets = get_binance_valid_trade_pairs_with_base_quote_assets()
 
-    for trade_pair_with_base_quote_asset in binance_valid_trade_pairs_with_base_quote_assets:
+    for trade_pair_with_base_quote_asset in trade_pairs_with_base_quote_assets:
         if trade_pair.upper() in trade_pair_with_base_quote_asset:
-            base_asset = binance_valid_trade_pairs_with_base_quote_assets[trade_pair_with_base_quote_asset]['baseAsset']
-            quote_asset = binance_valid_trade_pairs_with_base_quote_assets[trade_pair_with_base_quote_asset][
-                'quoteAsset']
+            base_asset = trade_pairs_with_base_quote_assets[trade_pair_with_base_quote_asset]['baseAsset']
```

Comment:
Can we just make
`base_asset = trade_pair_with_base_quote_asset['baseAsset']`?

Reply:
Not in this case, because trade_pairs_with_base_quote_assets is a dictionary of dictionaries like this {'ETHBTC': {'baseAsset': 'ETH', 'quoteAsset': 'BTC'}, ...} we need to find the user's trade pair match in the dictionary and from there get baseAsset and quoteAsset and format the string for the message

Reply:
Ok, then skip